dadi/lib/index.js

```diff
@@ -17,6 +17,8 @@ var raven = require('raven')
 var serveFavicon = require('serve-favicon')
 var serveStatic = require('serve-static')
 var session = require('express-session')
+var csrf = require('csurf')
```

Comment:
Bonus points (and a beer) are granted if you replace `var` with `const/let` as you go. ⭐️ 🍺

Reply:
@eduardoboucas I wasn't sure if we were doing that or if the plan was to stick with the existing style. Maybe it'd be worth tidying up the dependencies at least.